src/main/groovy/edu/oregonstate/mist/api/BuildInfoManager.groovy

```diff
@@ -0,0 +1,33 @@
+package edu.oregonstate.mist.api
+
+import io.dropwizard.lifecycle.Managed
+
+/**
+ * Created by georgecrary on 7/26/16.
+ */
+class BuildInfoManager implements Managed {
+    Info getInfo() {
+        info
+    }
+    private Info info = new Info()
+
+    @Override
+    public void start() throws Exception {
+        def stream = InfoResource.class.getResourceAsStream('/build.properties')
```

Comment:
That was the case in a previous commit. In the latest commit, that's not the case. Would you mind taking a look at these:
https://github.com/osu-mist/course-subjects-api/blob/feature/CO-468-subjects/src/main/groovy/edu/oregonstate/mist/api/InfoResource.groovy
https://github.com/osu-mist/course-subjects-api/blob/feature/CO-468-subjects/src/main/groovy/edu/oregonstate/mist/api/BuildInfoManager.groovy